try:
    value = input('Ingresa un número natural: ')
    print('El recíproco de', value, 'es', 1/int(value))        
except (ValueError, AttributeError):
    print('No se que hacer con', value)    
except ZeroDivisionError:
    #raise para pasar el error a la funcion padre
    print('La división entre cero no está permitida en nuestro Universo.')    
except:
    print('Ha sucedido algo extraño, ¡lo siento!')
else:
    print("fine")
finally:
    print("the end")
    
#AttributeError
#SyntaxError
#TypeError
#KeyboardInterrupt
#MemoryError
#OverflowError
